packages/graphql-proxy-server/.jest/setupTests.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+import '@testing-library/jest-dom/extend-expect';
+import { TextEncoder, TextDecoder } from 'util';
+
+import { toHaveNoViolations } from 'jest-axe';
+
+// Raise the default timeout from 5000
+jest.setTimeout(process.env?.CI ? 50_000 : 10_000);
```

Comment:
Isn't 50K way too much? I mean, something is wrong, if any process takes that much time. In unit tests every connection should be mocked and with a real connection, 50s wait is way too much.

Reply:
This is copy-pasted somewhere else. Can be removed on server tests